src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -156,6 +163,7 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
             brickRoadIndicator: hasGeneralSettingsError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
             routeName: SCREENS.WORKSPACE.PROFILE,
         },
+        ...(isCollectPolicy ? protectedCollectPolicyMenuItems : []),
```

Comment:
@lakchote I'm a bit confused about the naming of `protectedCollectPolicyMenuItems`. It seems to imply these items are only for "collect" policies, contrasting with `protectedMenuItems` which were restricted to **admin** visibility in the free plan. This naming could be misleading, as it suggests both sets are about access protection.

Would renaming it to `collectPolicyMenuItems` make it clearer that these items are specific to collect policies, not just protected ones? Or perhaps introducing a `shouldShowProtectedItems` flag could help clarify when these items should be displayed based on the user's role

In the context of deciding which menu items to show (isCollectPolicy, shouldShowProtectedItems), clear naming and logic are crucial for understanding and maintaining the code.

Reply:
You're right that it could lead to confusion.

I'm going to change it and will notify you when it's done.